code/modules/reagents/chemistry/machinery/chem_synthesizer.dm

```diff
@@ -5,44 +5,23 @@
 	icon_state = "dispenser"
 	amount = 10
 	resistance_flags = INDESTRUCTIBLE | FIRE_PROOF | ACID_PROOF | LAVA_PROOF
-	working_state = null
-	nopower_state = null
 	flags_1 = NODECONSTRUCT_1
+	use_power = NO_POWER_USE
 	var/static/list/shortcuts = list(
 		"meth" = "methamphetamine",
 		"tricord" = "tricordrazine"
 	)
-	var/mutable_appearance/top_overlay
-
-/obj/machinery/chem_dispenser/chem_synthesizer/Initialize()
-	. = ..()
-	GLOB.poi_list += src
```

Comment:
Only a botany/chem main would think to do this so if either got a scrying orb they'd know when to steal it / where to get it back from.

Reply:
pointless anyway since these are unobtainable

it's just a holdover from SCP-294